docs/TROUBLE.md

```diff
@@ -0,0 +1,162 @@
+
+Start your ground station software and connect to your Tx module by whatever method you have chosen.  You should see or hear the flight mode announced.  If parameter download also completes successfully, you can skip the next two topics
+
+This is a complex topic since there are so many ways to connect a Tx module to a ground station.
+
+- Use the [mLRS LUA script](LUA.md#usage) or [CLI](CLI.md) to check that Tx Ser Dest is correct and that Tx Ser Baudrate matches your wireless bridge or ground control setting.
+
+- If the connection is via a serial port, double check that rx is connected to tx, tx to rx, and ground to ground.
+
+- If your Tx module is the [R9M](FRSKY_R9.md#r9m-tx-module), don't forget to account for the inverted serial port and set dip switch 1 to on before powering up.
+
+- If the ground station runs on a mobile device and is connected via USB, an OTG adapter may be required.
+
+- If you want to connect via Bluetooth to the original [MatekSys Tx Module](MATEKSYS.md), don't forget to enable the HC-04 by moving all three [dip switches](https://www.mateksys.com/?portfolio=mr900-30-tx#tab-id-2) to the on position.
+
+- If the connection is via Bluetooth, the devices need to be paired first.
+
+- If the connection is via UDP over WiFi, connect to WiFi access point before starting the ground station.
+
+## 6. Confirm MAVLink message flow from Rx to flight controller
+
+Use the mLRS LUA script or another configuration method to temporarily set [Rx Snd RcChannel](PARAMETERS.md#rx-snd-rcchannel) to RC\_CHANNELS\_OVERRIDE.  If you use the mLRS LUA script there is no need to save; Just press the RTN button after selecting RC\_CHANNELS\_OVERRIDE.
+
+Connect a ground station via USB directly to your flight controller and view the MAVLink messages.
+With QGroundControl, use Analyze Tools -> MAVLink Inspector and select System 255 in the upper right.
+With Mission Planner, Press Ctrl-F, select MAVLink Inspector and expand Vehicle 255 -> Comp 68 -> RC\_CHANNELS\_OVERRIDE.
+
+You should see the RC\_CHANNELS\_OVERRIDE message is being received at the same rate as your configured mLRS mode.  Note that this also confirms your Tx module is receiving rc channel data from your radio and sending it to the Rx.
+
+- If you don't see these messages, check your wiring to ensure that the correct serial tx output on the Rx module is connected to the correct Rx input on your flight controller.
+
+## 7. Confirm MAVLink message flow from ground station to flight controller
+
+Using your ground station connected to your Tx module, verify that you can change flight modes (Using the ground station, not the radio) and successfully download parameters.
+
+- If you can't change flight modes or download parameters or issue other commands via the ground station, check your wiring to ensure that the correct serial tx output on your ground control device or wireless module or serial adapter is connected to the correct serial rx input on your Tx module.
```

Comment:
again ...

Reply:
I don't think we proved this wire is correct yet.